File: Task3/javascript/Subtask3/Task3.js
const readline= require('readline');
const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

rl.question("Enter width of diamond:"),(input) => {
    const n =parseInt(Input,10);

    const midd= Math.floor(n/2);

    for(let i=0;i <= mid-1; i >=0;i--){
        console.log(''.repeat(mid-1)+'*'.repeat(2*i+1));
    }
    for(let i=mid-1;i>=0;i--){
        console.log(''.repeat(mid-1)+'*'.repeat(2*i+1));
    }

    rl.close()

});

File: Task3/java/Subtask3/Task3.java
import java.util.Scanner;

public class Dia{
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter width of diamond:;");
        int n = scanner.nextInt();
        int mid =n/2;
        for (int i = 0; i <= mid; i++) {
            for (int j = 0; j < mid - i; j++) {
                System.out.print(" ");
    }
    for (int j = 0; j < 2 * i + 1; j++) {
        System.out.print("*");
    }
    System.out.println();

}
for (int i = mid - 1; i >= 0; i--) {
    for (int j = 0; j < mid - i; j++) {
        System.out.print(" ");
    }
    for (int j = 0; j < 2 * i + 1; j++) {
        System.out.print("*");
    }
    System.out.println();
}
        
scanner.close();
}
}
